src/commands/music/leave.rs

```diff
@@ -0,0 +1,26 @@
+use crate::primitives::Context;
+use anyhow::{Context as _, Result};
+use std::time::Instant;
+
+#[poise::command(prefix_command, slash_command)]
+/// Desconecta o bot do canal que você está conectado
+pub async fn leave(ctx: Context<'_>) -> Result<()> {
+    let guild = ctx.guild().context("No Guild!")?;
+
+    let ts = Instant::now();
+    let handler = ctx.say("Saindo...").await?;
```

Comment:
Isn't better defer interaction and send: `👌 Pronto.` as a ephemeral message?

Reply:
I was just following the pattern you used on the rest of the code. Sorry.